### ISSUE TITLE
Auto tunnel

### ISSUE THREAD

**Issue by ChristoferMinestar** [zeroeightysix/KAMI#106]
Preferably with autoeat support and a custom size

**Comment by blockparole** [comment]
Would be awesome if it could do diagonally also

**Comment by JerryWester** [comment]
Necroposting, doesn't Baritone/Fabritone have an auto-tunnel feature

**Comment by nothub** [comment]
yes it has indeed, i guess this ticket is outdated and could be closed.

**Comment by 5HT2** [comment]
We have a module in kami blue that lets you easily set the size with sliders instead of retyping the poor syntax for that command every time, I guess it's still valid?

**Comment by 5HT2** [comment]
> Would be awesome if it could do diagonally also

Baritone can't do this either, and it lacks proper liquid handling.

I guess this issue should instead be "Reliable autotunnel" :p
